### DISCUSSION TITLE
Move Item from My vault to Organization

### DISCUSSION THREAD

**Discussion by br0nes** [dani-garcia/vaultwarden#3666] [General]
I'm not sure if this is the right forum to post this. I have an issue with the client, where I'm moving an item from My vault to an Organization. When attempting to do so, it creates a clone instead of moving the item. It works properly in the web portal. The issue is only in the client. I'm not sure if that's the way it's supposed to work or an issue anyone else is experiencing. 

Client Version: 2023.5.1 (9424)
Vaultwarden Version: 1.28.1

**Comment by ghost** [comment]
To move an item, you have to want to modify it and at the bottom right you have a button to move it to your organisation and with the client and server up to date, it works very well for me without making a clone.

**Comment by br0nes** [comment]
Strange.. it worked this time but I had multiple items that didn't work this past week. It's working now.. Thanks for the help!
